tests/test_manager/test_manager.py

```diff
@@ -739,58 +737,44 @@ class TestMultiAgentManagerPackageConsistencyError:
-    - 'fetchai/fipa' of type protocol: the new version '0.16.0' conflicts with existing version '1.0.0' of the same package required by agents: [<fetchai/weather_station:0.27.0>]
-    - 'fetchai/ledger_api' of type protocol: the new version '0.13.0' conflicts with existing version '1.0.0' of the same package required by agents: [<fetchai/weather_station:0.27.0>]
-    - 'fetchai/oef_search' of type protocol: the new version '0.16.0' conflicts with existing version '1.0.0' of the same package required by agents: [<fetchai/weather_station:0.27.0>]
-    - 'open_aea/signing' of type protocol: the new version '0.13.0' conflicts with existing version '1.0.0' of the same package required by agents: [<fetchai/weather_station:0.27.0>]
-    - 'fetchai/state_update' of type protocol: the new version '0.13.0' conflicts with existing version '1.0.0' of the same package required by agents: [<fetchai/weather_station:0.27.0>]
-    """
-    )
+    EXPECTED_ERROR_MESSAGE = """cannot add project 'open_aea/my_first_aea:0.1.0': the following AEA dependencies have conflicts with previously added projects"""
 
     def setup(self):
         """Set up the test case."""
         self.project_public_id = MY_FIRST_AEA_PUBLIC_ID
         self.tmp_dir = TemporaryDirectory()
-        self.working_dir = os.path.join(self.tmp_dir.name, "MultiAgentManager_dir")
-        self.project_path = os.path.join(
-            self.working_dir, self.project_public_id.author, self.project_public_id.name
-        )
-        assert not os.path.exists(self.working_dir)
-        self.manager = MultiAgentManager(self.working_dir)
+        self.manager = MultiAgentManager(self.tmp_dir.name)
 
-    @pytest.mark.skip  # need remote registry
-    def test_run(self):
+    def test_run(self, *args):
         """
         Run the test.
 
-        First add agent project "fetchai/weather_station:0.27.0",
-        and then add "fetchai/weather_client:0.27.0".
+        First add agent project "open_aea/my_first_aea:0.1.0".
+        Then we mutate the stored `_package_id_prefix_to_version`
+        change the path and mock the `_versionless_projects_set`.
+        and attempt to add it anew, as if a different project.
         The second addition will fail because the projects
         contain conflicting AEA package versions.
         """
         self.manager.start_manager()
-        weather_station_id = PublicId.from_str("fetchai/weather_station:0.27.0")
-        self.manager.add_project(weather_station_id)
-        weather_client_id = PublicId.from_str("fetchai/weather_client:0.27.0")
+        my_first_aea_id = PublicId.from_str("open_aea/my_first_aea:0.1.0")
+        self.manager.add_project(my_first_aea_id)
+        base = Path(self.tmp_dir.name, "open_aea")
+        shutil.move(base / "my_first_aea", base / "my_first_aea_bumped")
+        p2v = self.manager._package_id_prefix_to_version
+        for k, (_, public_id_set) in p2v.items():
+            p2v[k] = ("0.0.0", public_id_set)
```

Comment:
since we do not have these packages in local repository I decided to "mock" it because - simpler and time efficient.